scripts/setup/tizen/install_tizen_sdk.sh

```diff
@@ -0,0 +1,358 @@
+        'libffi-devel-*.armv7l.rpm'
+        'liblzma-*.armv7l.rpm'
+        'libmount-devel-*.armv7l.rpm'
+        'libncurses6-*.armv7l.rpm'
+        'libreadline-*.armv7l.rpm'
+        'libuuid-*.armv7l.rpm'
+        'pcre-devel-*.armv7l.rpm'
+        'readline-devel-*.armv7l.rpm'
+        'xdgmime-*.armv7l.rpm')
+    download "-r -nd --no-parent $url" "${pkg_arr[@]}"
+
+    # Unified packages
+    url="  http://download.tizen.org/releases/milestone/tizen/unified/latest/repos/standard/packages/armv7l/"
+    pkg_arr=(
+        'aul-0*.armv7l.rpm'
+        'aul-devel-*.armv7l.rpm'
+        'bundle-0*.armv7l.rpm'
+        'bundle-devel-*.armv7l.rpm'
+        'buxton2-*.armv7l.rpm'
+        'cynara-devel-*.armv7l.rpm'
+        'dbus-1*.armv7l.rpm'
+        'dbus-devel-*.armv7l.rpm'
+        'dbus-libs-1*.armv7l.rpm'
+        'glib2-devel-2*.armv7l.rpm'
+        'json-glib-devel-*.armv7l.rpm'
+        'libcynara-client-*.armv7l.rpm'
+        'libcynara-commons-*.armv7l.rpm'
+        'libdns_sd-*.armv7l.rpm'
+        'libjson-glib-*.armv7l.rpm'
+        'libsessiond-0*.armv7l.rpm'
+        'libsystemd-*.armv7l.rpm'
+        'libtzplatform-config-*.armv7l.rpm'
+        'parcel-0*.armv7l.rpm'
+        'parcel-devel-*.armv7l.rpm'
+        'pkgmgr-info-*.armv7l.rpm'
+        'vconf-compat-*.armv7l.rpm'
+        'vconf-internal-keys-devel-*.armv7l.rpm')
+    download "-r -nd --no-parent $url" "${pkg_arr[@]}"
+
+    # Unified packages (snapshots)
+    url="http://download.tizen.org/snapshots/tizen/unified/latest/repos/standard/packages/armv7l/"
+    pkg_arr=(
+        'capi-network-nsd-*.armv7l.rpm'
+        'capi-network-thread-*.armv7l.rpm'
+        'libnsd-dns-sd-*.armv7l.rpm')
+    download "-r -nd --no-parent $url" "${pkg_arr[@]}"
+
+    # Install base sysroot
+    unzip -o '*.zip'
+    cp -rf data/* $TIZEN_SDK_ROOT
+
+    # Install packages
+    unrpm *.rpm
+    cp -rf lib usr $TIZEN_SDK_SYSROOT
+
+    # Make symbolic links relative TODO:Test if it's working
+    find "$TIZEN_SDK_SYSROOT"/usr/lib -maxdepth 1 -type l | while IFS= read -r -d '' pkg; do
+        ln -sf "$(basename "$(readlink "$pkg")")" "$pkg"
+    done
+    ln -sf ../../lib/libcap.so.2 $TIZEN_SDK_SYSROOT/usr/lib/libcap.so
+    ln -sf openssl1.1.pc $TIZEN_SDK_SYSROOT/usr/lib/pkgconfig/openssl.pc
+
+    # Cleanup remove tmp directory
+    rm -rf "${TMP_DIR:?}"
+
+    warning "You must add the appropriate environment variables before proceeding with matter."
+    echo "${COLOR_YELLOW}"
+    echo "export TIZEN_VESRSION=\"$TIZEN_VERSION\""
+    echo "export TIZEN_SDK_ROOT=\"$(realpath $TIZEN_SDK_ROOT)\""
+    echo "export TIZEN_SDK_TOOLCHAIN=\$TIZEN_SDK_ROOT/tools/arm-linux-gnueabi-gcc-9.2\""
+    echo "export TIZEN_SDK_SYSROOT=\"\$TIZEN_SDK_ROOT/platforms/tizen-6.0/mobile/rootstraps/mobile-6.0-device.core\""
+    echo "export PATH=\"\$TIZEN_SDK_ROOT/tools/ide/bin:\$TIZEN_SDK_ROOT/tools:\$PATH\""
+    echo -n "${COLOR_NONE}"
+
+}
+
+# Check if the script is run with elevated privileges
```

Comment:
I don't think that this check is required. There shall be no side effects if this scrip is run as root/sudo.